core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -4,22 +4,30 @@
 <!--[if gte IE 9]><!-->
 <!-- jquery.js, angular.js and jquery-ui.js are removed from bundled js because they need to be at the header. Including
 bundled js at the header will block rendering.-->
+
   <script src="/third_party/static/jquery-2.1.1/jquery.min.js">
   </script>
 <!--<![endif]-->
-<script src="/third_party/static/jqueryui-1.10.3/jquery-ui.min.js">
-</script>
-<script src="/third_party/static/angularjs-1.4.7/angular.min.js"></script>
-<script src="/third_party/static/jquery-ui-touch-punch-0.3.1/jquery.ui.touch-punch-improved.js"></script>
+  <script src="/third_party/static/jqueryui-1.10.3/jquery-ui.min.js">
+  </script>
+  <script src="/third_party/static/angularjs-1.4.7/angular.min.js"></script>
+  <script src="/third_party/static/jquery-ui-touch-punch-0.3.1/jquery.ui.touch-punch-improved.js"></script>
 <!-- See http://docs.mathjax.org/en/latest/start.html#mathjax-cdn -->
-<script type="text/x-mathjax-config">
+  <script type="text/x-mathjax-config">
   MathJax.Hub.Config({
-    skipStartupTypeset: true,
-    messageStyle: 'none',
-    'HTML-CSS': {
-      showMathMenu: false
-    }
+    CommonHTML: { linebreaks: { automatic: true } },
+    "HTML-CSS": { linebreaks: { automatic: true } },
```

Comment:
Jst one more doubt !!!Should I close this pull request and after the changes that u have told me to do ,open a new pull request ??